index.js

```diff
@@ -0,0 +1,105 @@
+console.log("I'm ready!");
+
+//Iteration 1
+
+//1.1
+let hacker1 = "Lucia";
+
+//1.2
+console.log(`The driver's name is ${hacker1}`);
+
+//1.3
+let hacker2 = "Sarah";
+
+//1.4
+console.log(`The navigator's name is ${hacker2}`);
+
+
+//Iteration 2
+
+//2.1
+if(hacker1.length>hacker2.length){
+console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+} else if (hacker2.length>hacker1.length){
+console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+} else {
+console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+};
+
+//Iteration 3
+
+//3.1
+let str = "";
+
+for (let i = 0; i<hacker1.length; i++) {
+ str += hacker1[i].toUpperCase() + " " ;
+};
+console.log(str);
+
+//3.2 (why 'undefined' before name??)
+let str2 = "";
+for (let i = hacker2.length--; i>=0; i--) {
+ str2 += hacker2[i] + " ";
+};
+console.log(str2);
+
+//3.3
+if(hacker1.charAt(0)<hacker2.charAt(0)) {
+console.log("The driver's name goes first.")
+} else if (hacker2.charAt(0)<hacker1.charAt(0)) {
+console.log("Yo, the navigator goes first definitely.")
+} else {
+console.log("What?! You both have the same name?")
+};
```

Comment:
Only compares the first character of the string, so considers that both have the same names if they start with the same letter. You could do include those condition statements inside a for loop, and check every character (like Uros showed this morning during review, don't hesitate if you have any question about it).